code/obj/item/clothing/glasses.dm

```diff
@@ -366,6 +372,13 @@
 		if (pinhole)
 			block_eye = null
 
+/obj/item/clothing/glasses/twopatch //this is so stupid I love it
+	name = "double eyepatches"
+	desc = "Double the eyepatches, double the coolness."
+	icon_state = "twopatch"
+	item_state = "radio"
```

Comment:
radio?

Reply:
it's what the normal eyepatch uses. It works for now